src/components/app/app.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { fireEvent, render, screen, within } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import { App } from './app'
+import React from 'react'
+import * as deck from '../../services/deck'
+import { CardSuit, CardValue } from '../../interfaces/card.interface'
+
+test('Renders app', () => {
+    render(<App />)
+
+    expect(screen.getByText('DEALER')).toBeInTheDocument
```

Comment:
on this line and next we are not invoking the matcher of `toBeInTheDocument`. Thus, it is not asserting that the element exists. You can prove this by just adding `.not` before `.toBeInTheDocument` and the test _still_ passes.

Update each of these to invoke the matcher function:

```
    expect(screen.getByText('DEALER')).toBeInTheDocument()
    expect(screen.getByText('PLAYER')).toBeInTheDocument()
```